BSP/sharpMemoryLCD/LS013B7DH03.c

```diff
@@ -50,6 +50,27 @@ static void LCDdisplayEnable(bool en)
     }
 }
 
+/**
+ * @brief adapt the bit and byte order so thats screen prints everything in expected direction
+ * Bits need to be reversed in all bytes for this screen.
+ * And everything needs to be rotated because screen is upside down.
+ *
+ * @param pixelBuf display buffer where one bit is one pixel.
+ * @param nbBytePixelAsciiLine length of buffer. This must be multiple of 2
+ */
+static void reorderBitsToScreen(uint8_t *pixelBuf, uint16_t nbBytePixelAsciiLine)
+{
+    //TODO this could be optimized doing 4 bytes at a time (with __REV instruction)
+    // check length is multiple of 2
+    assert_param((nbBytePixelAsciiLine%2) == 0);
```

Comment:
Does it make sense to make an if statement and do nothing if not multiple ?

Reply:
assert is enough I think. It's a sanity check but it's always the case when you print full lines. And this screen requires to print full lines anyway